include/util/coordinate.hpp

```diff
@@ -40,7 +40,7 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 namespace osrm
 {
 
-constexpr const double COORDINATE_PRECISION = 1e6;
+constexpr const double COORDINATE_PRECISION = 1e7;
```

Comment:
What makes you think this could break user code? Wouldn't you see the actual encoding as internal state?